canvas_cli/apps/plugin/plugin.py

```diff
@@ -133,6 +138,29 @@ def init(
     print.json(f"Project created in {project_dir}", project_dir=project_dir)
 
 
+@app.command(short_help="Validates the Canvas Manifest json file of a plugin package")
+def validate(
+    package: Path = typer.Argument(
```

Comment:
Since `validate` is used so commonly and this function does something very specific, it might be better to call it `validate_manifest_json` or something similar.